File: back/Web2Backend/WebApplication1/Models/Proizvod.cs
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace WebApplication1.Models
{


    [Table("Proizvod")]
    public class Proizvod
    {
        [Column("Ime")]
        [Required]
        public string Ime { get; set; }

        [Column("Cena")]
        [Required]
        public double Cena { get; set; }

        [Column("Id")]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        [Key]
        public int Id { get; set; }
    
        public virtual List<Sastojak> Sastojaks { get; set; }

       

    }

  
    
}

File: web-app/src/app/Entities/proizvod.ts
import { Sastojak } from "./sastojak";

export class Proizvod {
    ime : string;
    cena : number;
    id : number;
    sastojaks : Sastojak[]

    constructor(ime : string,cena:number,id:number,sastojaks : Sastojak[])
    {
        this.ime = ime;
        this.cena = cena;
        this.id = id;
        this.sastojaks = sastojaks;
    }
}
